Exec/radiation_tests/RadBreakout/filt_prim.H

```diff
@@ -0,0 +1,117 @@
+                                ff3(3,S) * (Stmp(i-3,j,k,URHO ) + Stmp(i+3,j,k,URHO ));
+            Snew(i,j,k,UMX  ) = ff3(0,S) *  Stmp(i  ,j,k,UMX  ) +
+                                ff3(1,S) * (Stmp(i-1,j,k,UMX  ) + Stmp(i+1,j,k,UMX  )) +
+                                ff3(2,S) * (Stmp(i-2,j,k,UMX  ) + Stmp(i+2,j,k,UMX  )) +
+                                ff3(3,S) * (Stmp(i-3,j,k,UMX  ) + Stmp(i+3,j,k,UMX  ));
+            Snew(i,j,k,UTEMP) = ff3(0,S) *  Stmp(i  ,j,k,UTEMP) +
+                                ff3(1,S) * (Stmp(i-1,j,k,UTEMP) + Stmp(i+1,j,k,UTEMP)) +
+                                ff3(2,S) * (Stmp(i-2,j,k,UTEMP) + Stmp(i+2,j,k,UTEMP)) +
+                                ff3(3,S) * (Stmp(i-3,j,k,UTEMP) + Stmp(i+3,j,k,UTEMP));
+        }
+        else if (filt_T == 4) {
+            filtered = true;
+            Snew(i,j,k,URHO ) = ff4(0,S) *  Stmp(i  ,j,k,URHO ) +
+                                ff4(1,S) * (Stmp(i-1,j,k,URHO ) + Stmp(i+1,j,k,URHO )) +
+                                ff4(2,S) * (Stmp(i-2,j,k,URHO ) + Stmp(i+2,j,k,URHO )) +
+                                ff4(3,S) * (Stmp(i-3,j,k,URHO ) + Stmp(i+3,j,k,URHO )) +
+                                ff4(4,S) * (Stmp(i-4,j,k,URHO ) + Stmp(i+4,j,k,URHO ));
+            Snew(i,j,k,UMX  ) = ff4(0,S) *  Stmp(i  ,j,k,UMX  ) +
+                                ff4(1,S) * (Stmp(i-1,j,k,UMX  ) + Stmp(i+1,j,k,UMX  )) +
+                                ff4(2,S) * (Stmp(i-2,j,k,UMX  ) + Stmp(i+2,j,k,UMX  )) +
+                                ff4(3,S) * (Stmp(i-3,j,k,UMX  ) + Stmp(i+3,j,k,UMX  )) +
+                                ff4(4,S) * (Stmp(i-4,j,k,UMX  ) + Stmp(i+4,j,k,UMX  ));
+            Snew(i,j,k,UTEMP) = ff4(0,S) *  Stmp(i  ,j,k,UTEMP) +
+                                ff4(1,S) * (Stmp(i-1,j,k,UTEMP) + Stmp(i+1,j,k,UTEMP)) +
+                                ff4(2,S) * (Stmp(i-2,j,k,UTEMP) + Stmp(i+2,j,k,UTEMP)) +
+                                ff4(3,S) * (Stmp(i-3,j,k,UTEMP) + Stmp(i+3,j,k,UTEMP)) +
+                                ff4(4,S) * (Stmp(i-4,j,k,UTEMP) + Stmp(i+4,j,k,UTEMP));
+        }
+    }
+
+    if (filtered) {
+
+        Snew(i,j,k,URHO ) = amrex::max(Snew(i,j,k,URHO ), castro::small_dens);
+        Snew(i,j,k,UTEMP) = amrex::max(Snew(i,j,k,UTEMP), castro::small_temp);
+
+        eos_t eos_state;
+
+        eos_state.rho = Snew(i,j,k,URHO);
+        eos_state.T = Snew(i,j,k,UTEMP);
+
+        Real rhotmpInv = 1.0_rt / Stmp(i,j,k,URHO);
+        for (int n = 0; n < NumSpec; ++n) {
+            eos_state.xn[n] = Stmp(i,j,k,UFS+n) * rhotmpInv;
+        }
+#if NAUX_NET > 0
+        for (int n = 0; n < NumAux; ++n) {
+            eos_state.aux[n] = Stmp(i,j,k,UFX+n) * rhotmpInv;
+        }
+#endif
+
+        eos(eos_input_rt, eos_state);
+
+        Snew(i,j,k,UEINT) = Snew(i,j,k,URHO) * eos_state.e;
+        Snew(i,j,k,UEDEN) = Snew(i,j,k,UEINT) + 0.5_rt *
+                                                (Snew(i,j,k,UMX) * Snew(i,j,k,UMX)) /
```

Comment:
our convention is to always include all 3 components of momentum in the kinetic energy